nvdaHelper/remote/winword.cpp

```diff
@@ -945,7 +963,8 @@ void detectAndGenerateColumnFormatXML(IDispatchPtr pDispatchRange, wostringstrea
 	}
 	xmlStream <<L"text-column-number=\"" << columnNumber << "\" ";
 
-	// Finally, double check that we calculated the full width of the document
+	/*
+	The following commented out code to the end of this function can be used to double check that we calculated the full width of the document

```

Comment:
Rather than commenting this out, would it make sense to add an additional nvdaHelperDebugFlag?  May be you've already thought about this and thought it wouldn't be worth it. That's ok, I just wanted to bring up the idea.

Reply:
LOG_DEBUG is defined to nothing by default, I.e. none of those tokens within the brackets will be compiled, unless nvdahelperLogLevel is debug.